clojure-mode.el

```diff
@@ -195,6 +195,11 @@ Out-of-the box clojure-mode understands lein, boot and gradle."
   (let ((map (make-sparse-keymap)))
     (define-key map (kbd "C-:") #'clojure-toggle-keyword-string)
     (define-key map (kbd "C-c SPC") #'clojure-align)
+    (define-key map (kbd "C-c th") #'clojure-thread)
```

Comment:
They should all be a single key under a 2-key predix. For instance, I would suggest the keys be `t`, `u`, `f`, `l`, and `a`. And the prefix could be `C-c C-r` or `C-c t` or something else even.